nomos-da/full-replication/src/attestation.rs

```diff
@@ -0,0 +1,52 @@
+use bytes::Bytes;
+use nomos_core::{
+    da::{attestation, auth::Signer, blob},
+    wire,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::{hash, Blob, Voter};
+
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
+#[cfg_attr(feature = "openapi", derive(utoipa::ToSchema))]
+pub struct Attestation {
+    blob: [u8; 32],
```

Comment:
Ah, I just found that the size of blob is going to be always 32 bytes as long as we use BLS12-381 curve.

Reply:
> If the size of blob is much bigger than 32 bytes, could we probably include hash instead of blob in the Attestation in the future?

The naming here is bit misleading, because when attestation is created, protocol provides blob hash to construct `Attestation` struct:
https://github.com/logos-co/nomos-node/blob/6ad88566bddfdb777210a86b363fe355eb18677c/nomos-da/full-replication/src/lib.rs#L205-L208

Reply:
Yeah. I understand. Thank you!